protocols/bgp/packet/path_attributes.go

```diff
@@ -817,6 +817,17 @@ func PathAttributes(p *route.Path, iBGP bool, rrClient bool) (*PathAttribute, er
 	last.Next = nextHop
 	last = nextHop
 
+	if p.BGPPath.MED != 0 {
```

Comment:
Do we have a testcase regarding the proper rendering with MED?

Reply:
No.